dev-client/src/screens/SiteSettingsScreen/SiteSettingsScreen.tsx

```diff
@@ -91,7 +91,7 @@ export const SiteSettingsScreen = ({siteId}: Props) => {
             size="container"
             alignItems="center"
             justifyContent="space-around">
-            <Icon name="people" />
+            <Icon name="people-alt" />
```

Comment:
Issue: Why this change? As far as I can tell we use the "people" icon for the badges. But also this is commented out.

Reply:
I checked the Figma file, and it looked closest to `people-alt`. Prior to this PR, we had been using a combination of both.